castor/docker_events/admin.py

```diff
@@ -0,0 +1,8 @@
+from django.contrib import admin
+
+from docker_events.models import DockerEvent
+
+
+class DockerEventAdmin(admin.ModelAdmin):
+    pass
+admin.site.register(DockerEvent, DockerEventAdmin)
```

Comment:
You could use the [`@admin.register`](https://docs.djangoproject.com/en/1.10/ref/contrib/admin/#django.contrib.admin.register) decorator to avoid running code in this way. That's more like a personal preference though.